### PR TITLE
doc: Improve Windows native build instructions

### PR DESCRIPTION
**Pull request by murrayn**
The current instructions assume usage of `PowerShell`; running in a `Command Prompt` results in a syntax error. Proposed change works with both `Command Prompt` and `PowerShell`. 

Also a few other minor formatting changes.

### CODE REVIEW

File: build_msvc/README.md [hunk 1]
```diff
@@ -31,20 +31,13 @@ Additional dependencies required from the [bitcoin-core](https://github.com/bitc
...
 
 ```
-    PS >.\vcpkg install boost:x64-windows-static `
-    libevent:x64-windows-static `
-    openssl:x64-windows-static `
-    zeromq:x64-windows-static `
-    berkeleydb:x64-windows-static `
-    secp256k1:x64-windows-static `
-    leveldb:x64-windows-static
+    PS >.\vcpkg install boost:x64-windows-static libevent:x64-windows-static openssl:x64-windows-static zeromq:x64-windows-static berkeleydb:x64-windows-static secp256k1:x64-windows-static leveldb:x64-windows-static
```
Comment (ken2812221) [inline build_msvc/README.md]: Could use `.\vcpkg install --triplet x64-windows-static boost-filesystem boost-signals2 boost-test libevent openssl zeromq berkeleydb secp256k1 leveldb` instead. Don't have to install all boost dependencies.

**Review by ken2812221 — COMMENTED** [review]
Concept ACK

**Comment by sipsorcery** [comment]
tACK.

@ken2812221 I ran an appveyor build with each of the two`vcpkg` commands below and both dragged in the exact same 80 packages, 70 of which were boost. 

Your version is an improvement as it's shorter and I guess if the Boost dependency chain ever changes it could end up pulling in less packages.

`vcpkg install --triplet x64-windows-static boost-filesystem boost-signals2 boost-test libevent openssl zeromq berkeleydb secp256k1 leveldb`

`vcpkg install boost:x64-windows-static libevent:x64-windows-static openssl:x64-windows-static zeromq:x64-windows-static berkeleydb:x64-windows-static secp256k1:x64-windows-static leveldb:x64-windows-static`

File: build_msvc/README.md (hunk 1 above)
Comment (sipsorcery) [reply build_msvc/README.md]: +1 for switching to @ken2812221's vcpkg command (see note below).

**Comment by murrayn** [comment]
I've incorporated @ken2812221 's suggestion.

**Comment by fanquake** [comment]
utACK 5237850

**Pull request merged**